verify_post_gig.py

```
@@ -0,0 +1,198 @@
+        if state.get('currentScene') == 'INTRO':
+            print("Skipping INTRO...")
+            page.mouse.click(100, 100)
+            time.sleep(2)
+            state = page.evaluate("window.gameState || {}")
+            print(f"Current Scene (After Skip): {state.get('currentScene')}")
+
+        try:
+            # Try to click LOAD GAME
+            try:
+                page.wait_for_selector("button:has-text('LOAD GAME')", timeout=3000)
+            except:
```

Ruff (0.15.2)

[error] Do not use bare `except`

(E722)